044-Dapr/Coach/Solution-08.md

```diff
@@ -143,7 +143,7 @@ Now that your container images have been uploaded to the Azure Container Registr
 
     > IMPORTANT: The Azure Container Registry has the **admin** account enabled to make this demo easier to deploy (doesn't require the deployer to have Owner access to the subscription or resource group the Azure resources are deployed to). **This is not best practice!** In a production deployment, use a managed identity or service principal to authenticate between the AKS cluster & the ACR. See the [documentation](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-authentication?tabs=azure-cli) for more about the options and how to set up.
 
-1. 	Deploy your new services to AKS. Navigate to the `src/k8s/components` directory and run the following:
+1. 	Deploy your new services to AKS. Navigate to the `Resources/k8s/components` directory and run the following:
```

Comment:
I think it should be 'Resources/dapr/components', please verify

Reply:
Yes, fixed in two places.